packages/styles/scss/components/tableofcontents/_tableofcontents.scss

```diff
@@ -46,6 +46,11 @@
 
 @mixin tableofcontents {
   .#{$prefix}--tableofcontents {
+    text-size-adjust: 100%;
+    -webkit-text-size-adjust: 100%;
+    -ms-text-size-adjust: 100%;
+    -moz-text-size-adjust: 100%;
```

Comment:
@diegomarcuz Are these prefixes needed?

Reply:
Yeah Carbon (and I believe our library) requires `autoprefixer` in place for Sass/CSS build chain.

Reply:
ohhh got it, I thank you @asudoh, does it occurs even on dev mode ?

Reply:
Not certain, but it's a requirement of Carbon.

Reply:
Because on my dev mode, It doesn't occur. When I remove the `webkit` prefix the property doesn't work. @asudoh

Reply:
Would you try looking at the Storybook build config and make sure it's configured with the latest `autoprefixer`?

Reply:
it is what I found, is it right ?

<img width="649" alt="Screen Shot 2020-08-24 at 09 25 44" src="https://user-images.githubusercontent.com/37422384/91044518-cac1df80-e5eb-11ea-87e9-141a548773c0.png">

Reply:
Does the browserslist setting cover the browsers you are dealing with? Also ref: https://github.com/postcss/autoprefixer/issues/264

Reply:
Checking, because I am not so familiar with `autoPrefixer`.

Reply:
```suggestion
```

With #3706, those style rules will be generated automatically.

Reply:
@asudoh, Thanks so much for your help, I removed the prefixes.